examples/machine_translation/README.md

```diff
@@ -0,0 +1,48 @@
+# English-Spanish machine translation with keras-nlp
+
+This example will show how to train a Transformer-based machine translation 
+model using APIs provided by Keras-NLP. This instruction shows how to train the 
+model, and evaluate with customized English sentences.
+
+## Installing dependencies
+
+Pip dependencies for all keras-nlp examples are listed in `setup.py`. To install
+both the keras-nlp library from source and all other dependencies required to
+run the example, run the below command. You may want to install to a self
+contained environment (e.g. a container or a virtualenv).
+
+```shell
+pip install -e ".[examples]"
+```
+
+## Train the machine translation model and save to disk
+
+At the root directory of keras-nlp, run the following command:
+
+```shell
+python ./examples/machine_translation/train.py \
+    --num_epochs=3 \
+    --saved_model_path="saved_models/machine_translation"
+```
+
+If it finishes successfully, you should see your console prints out the 
+following information:
+```bash
```

Comment:
I don't think you want bash syntax highlighting. Just do a normal codeblock ```

Reply:
Thanks, done!